routers/web/web.go

```diff
@@ -1247,6 +1247,7 @@ func registerRoutes(m *web.Route) {
 		m.Group("/contributors", func() {
 			m.Get("", repo.Contributors)
 			m.Get("/{contribution_type}", repo.Contributors)
+			m.Get("/data", repo.ContributorsData)
 		}, context.RepoRef(), repo.MustBeNotEmpty, context.RequireRepoReaderOr(unit.TypePullRequests, unit.TypeIssues, unit.TypeReleases))
```

Comment:
Can I delete `context.RequireRepoReaderOr(unit.TypePullRequests, unit.TypeIssues, unit.TypeReleases)` completely? How should we handle this?

If you ask why that line is there, I would say because I copied Activity endpoint and only changed relevant bits so that I will be sure my impl will work.

Reply:
I don't think you should delete them.